### PR TITLE
updated with network switch

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Added network switch and added to the friendly bot.

### CODE REVIEW

File: src/components/network/Network.tsx [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 }
 
-const Network = ({networkProvider, network, onNetworkChange}: Props) => {
+const Network = ({network, onNetworkChange}: Props) => {
   function handleChangeNetwork(e: React.ChangeEvent<HTMLInputElement>) {
     onNetworkChange(e.target.value);
   }
 
+  const networks = {
+    "TESTNET": "wss://testnet-node-1.cere.network:9944",
```
Comment (evgeny-s) [inline src/components/network/Network.tsx:14]: Let's use enum here.
Also move it to the separate file

File: src/components/network/Network.tsx [hunk 2]
```diff
@@ -19,20 +24,20 @@ const Network = ({networkProvider, network, onNetworkChange}: Props) => {
           className="btn-check btn-outline-primary"
           name="options"
           id="testnet"
-          value="wss://testnet-node-1.cere.network:9944"
+          value="TESTNET"
```
Comment (evgeny-s) [inline src/components/network/Network.tsx:27]: Use enum here

File: src/components/network/Network.tsx [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
 
+  const networks = {
+    "TESTNET": "wss://testnet-node-1.cere.network:9944",
+    "TESTNET_DEV": "wss://testnet-node-1.dev.cere.network:9944",
+    "TESTNET_DEV1": "wss://testnet-node-1.dev1.cere.network:9944"
+  }
+
   return (
     <>
       <div className="tab d-flex justify-content-end m-3  ">
```
Comment (evgeny-s) [inline src/components/network/Network.tsx:21]: Use `.map` rendering here:
```
enumKeys.map((item) => (
   <>
     <input itemValue />
     <label>itemLabel</label>
   </>
))
```

File: src/modules/network/reducers.ts [hunk 4]
```diff
@@ -4,22 +4,19 @@ import {Action, handleActions} from 'redux-actions';
...
 export interface NetworkState {
-  networkProvider: string;
   network: string;
-
 }
 
 const initialState: NetworkState = {
-  networkProvider: 'wss://testnet-node-1.cere.network:9944',
-  network: 'Testnet'
+  network: 'TESTNET',
```
Comment (evgeny-s) [inline src/modules/network/reducers.ts:11]: Use enum here

File: src/services/LaboratoryApiService.ts [hunk 5]
```diff
@@ -48,8 +48,8 @@ class LaboratoryApiService implements LaboratoryApiServiceInterface {
     return block;
 
   }
-  postFriendBotAssetRequest = async (destination: string) => {
-    return (await this.httpClient.post(`/friend-bot/request-assets`, {destination})).data;
+  postFriendBotAssetRequest = async (destination: string, network: string) => {
```
Comment (evgeny-s) [inline src/services/LaboratoryApiService.ts:51]: string -> enum

**Review by evgeny-s — CHANGES_REQUESTED** [review]

File: src/components/network/Network.tsx (hunk 1 above)
Comment (karthikeyan-cere) [reply src/components/network/Network.tsx]: Fixed

File: src/components/network/Network.tsx (hunk 2 above)
Comment (karthikeyan-cere) [reply src/components/network/Network.tsx]: Fixed

File: src/modules/network/reducers.ts (hunk 4 above)
Comment (karthikeyan-cere) [reply src/modules/network/reducers.ts]: Fixed

File: src/services/LaboratoryApiService.ts (hunk 5 above)
Comment (karthikeyan-cere) [reply src/services/LaboratoryApiService.ts]: Fixed

File: src/components/network/Network.tsx (hunk 3 above)
Comment (karthikeyan-cere) [reply src/components/network/Network.tsx]: fixed.

File: src/components/network/network.enum.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export enum Networks {
```
Comment (evgeny-s) [inline src/components/network/network.enum.ts:1]: Do we need this?

File: src/components/network/networkProvider.enum.ts [hunk 7]
```diff
@@ -0,0 +1,6 @@
+export enum NetworksProvider {
```
Comment (evgeny-s) [inline src/components/network/networkProvider.enum.ts:1]: Do we need this?

**Review by evgeny-s — CHANGES_REQUESTED** [review]

File: src/components/network/network.enum.ts (hunk 6 above)
Comment (karthikeyan-cere) [reply src/components/network/network.enum.ts]: deleted.

File: src/components/network/networkProvider.enum.ts (hunk 7 above)
Comment (karthikeyan-cere) [reply src/components/network/networkProvider.enum.ts]: deleted.

**Review by evgeny-s — APPROVED** [review]

**Pull request merged**